packages/module/src/utils/svg-utils.ts

```diff
@@ -97,23 +97,23 @@ export const pointTuplesToPath = (segments: PointTuple[]): string => {
   return pathSegments.join(' ');
 };
 
-// Returns the SVG path data string representing the polygon, expanded and rounded.
+// Returns the SVG path data string representing the polygon and it's segments if available, expanded and rounded.
 /**
  * @param polyPoints
  * @param hullPadding
  */
-export function hullPath(polyPoints: PointTuple[], hullPadding: number | HullPaddingGetter = 0): string {
+export function hullPath(polyPoints: PointTuple[], hullPadding: number | HullPaddingGetter = 0): { path: string, segments?: PointTuple[][] } {
```

Comment:
Changing the return for an exported function is a breaking change. We could either add a new function or this would have to go into V5.

Reply:
sure @jeff-phillips-18, let's keep `hullPath` as is https://github.com/patternfly/react-topology/pull/30/commits/dc075745819830b24c415e859f04b23ef558c6b2

Only `hullPathAndSegments` function will return both path and segments